accepted/0000-notification-system-for-cli-updates.md

```diff
@@ -0,0 +1,31 @@
+# Notification system for cli updates
+
+## Summary
+
+A new notification system to warn users about new versions of the **npm cli**.
+
+## Motivation
+
+- Remove an extra http request from the cli
```

Comment:
I am not sure this makes sense. It is just one request per week. During an installation there are hundreds of requests.

Reply:
my guess is that when this was brought up it was with regards to commands that should not necessarily hit the registry, such as `npm ls` but i do agree with your point, it might be silly and not worth having this listed in the RFC after all

Reply:
I think it's worth it to keep it if reducing HTTP calls where they may not be necessary is a goal of the project (I think that's a fair and valid goal).